lib/logstash/inputs/stdin.rb

```diff
@@ -12,14 +13,18 @@ class LogStash::Inputs::Stdin < LogStash::Inputs::Base
 
   default :codec, "line"
 
-  public
+  def initialize(*args)
+    super(*args)
+    @stop_requested = Concurrent::AtomicBoolean.new(false)
+  end
```

Comment:
This is the only plugin using the #initialize method, to be consistent we should move that to the #register method?

(don't get me wrong here I would love to see us use the #initialize method instead of register)